carbon-design-system/carbon-addons-iot-react
# fix(table): add tooltip to search icon in toolbar

Closes #3731 

**Summary**

- Add Carbon tooltip to search icon in table toolbar

**Change List (commits, features, bugs, etc)**

- Add wrapper to `TableSearch`
- Add conditional render for Carbon tooltip
- **NOTE:** updated 3 cypress image snapshots that are not relevant to PR updates. Seems like unexpected hover events are fired in the table.

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3735--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-searching)
- Hover over search icon in table toolbar
- Verify that tooltip appeared
- Enable RTL mode
- Change any knob to reload Table
- Hover over search icon in table toolbar

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
